resources/kprove.mak

```diff
@@ -57,6 +57,9 @@ TANGLER:=$(PANDOC_TANGLE_SUBMODULE)/tangle.lua
 LUA_PATH:=$(PANDOC_TANGLE_SUBMODULE)/?.lua;;
 export LUA_PATH
 
+K_OPTS:=-Xmx12g
+export K_OPTS
+
```

Comment:
would it work for a machine that doesn't have enough memory?

Reply:
@denis-bogdanas please use `?=` instead of `:=`, so that we can adjust when we run `make`.

that is:
```
K_OPTS?=-Xmx12g
```
instead of:
```
K_OPTS:=-Xmx12g
```

Reply:
It possibly would, but any error in specs leads to extra paths and that leads to OOMError and hard to debug issue.

Corrected K_OPTS.